addons/point_of_sale/models/product.py

```diff
@@ -63,7 +63,7 @@ def _load_pos_data_fields(self, config_id):
         return [
             'id', 'display_name', 'lst_price', 'standard_price', 'categ_id', 'pos_categ_ids', 'taxes_id', 'barcode', 'name',
             'default_code', 'to_weight', 'uom_id', 'description_sale', 'description', 'product_tmpl_id', 'tracking', 'type', 'service_tracking', 'is_storable',
-            'write_date', 'available_in_pos', 'attribute_line_ids', 'active', 'image_128', 'combo_ids', 'product_template_variant_value_ids',
+            'write_date', 'available_in_pos', 'attribute_line_ids', 'active', 'image_128', 'combo_ids', 'product_template_variant_value_ids', 'public_description',
```

Comment:
Find a way to load this fields only in self_order not in point_of_sale

Reply:
@davidmonnom wdyt ?

Reply:
Oh yes :pray:

Reply:
@davidmonnom wdyt ?

Reply:
@davidmonnom can you take a look ?